### ISSUE TITLE
Feature request: Checkbox Count parallell activities only once

### ISSUE THREAD

**Issue by eiriktorheim** [aronwoost/google-calendar-hours#36]
Hi, 

I have a feature request that would make the application much more useful for me. 

I want to check how many hours I worked one week, and when I do check with my work calendar, sometimes I have multiple parallell events making the total hours for the week go more than actually time spent. 

Would be great if there was an option to count paralell activites only one time. 

Eirik

**Comment by aronwoost** [comment]
You are right! I actually think this should be the default. Because: how can a day have more then 24h?? 🧐😇

Fair warning: I just cleaned up the Github issues. So it'll probably take another year or two until this feature is implemented... 😄
